### PR TITLE
AMM/CLOB Clarification

### PR DESCRIPTION
**Pull request by oeggert**
Adding conceptual doc and diagram to highlight AMM integration with XRPL DEX.

### CODE REVIEW

File: docs/concepts/tokens/decentralized-exchange/automated-market-makers.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ Tokens with different issuers are considered different assets. This means that t
 
 When the flow of funds between the two assets in a pool is relatively active and balanced, the fees provide a source of passive income for liquidity providers. However, when the relative price between the assets shifts, liquidity providers can take a loss on the [currency risk](https://www.investopedia.com/terms/c/currencyrisk.asp).
 
+### DEX Interaction
+
+AMMs are integrated with the central limit order book (CLOB)-based DEX to enhance liquidity on the XRPL. Offers and payments are automatically optimized to determine whether swapping within a liquidity pool, through the order book, or both, provides the best rate and executes accordingly. This ensures that transactions use the most efficient path for trades, whether through offers on the DEX or through AMM pools, or a combination of the two.
+
+The diagram below illustrates how an offer interacts with other offers and AMM liquidity in the XRPL DEX.
+
+![Offer path through XRPL DEX.](/docs/img/amm-clob-diagram.png)
```
Comment (mDuo13) [inline docs/concepts/tokens/decentralized-exchange/automated-market-makers.md]: You don't really need to mention that it's the XRPL when it's documentation on xrpl.org.

```suggestion
AMMs are integrated with the central limit order book (CLOB)-based DEX to enhance liquidity. Offers and payments are automatically optimized to determine whether swapping within a liquidity pool, through the order books, or both, provides the best rate and executes accordingly. This ensures that transactions use the most efficient path for trades, whether through offers on the DEX or through AMM pools, or a combination of the two.

The diagram below illustrates how an offer interacts with other offers and AMM liquidity in the DEX.

![Offer path through DEX.](/docs/img/amm-clob-diagram.png)
```

**Review by mDuo13 — APPROVED** [review]

**Comment by tequdev** [comment]
@oeggert @mDuo13 

I just checked the diagram and this needs to be fixed.

Why does this diagram have the Ripple logo?

**Comment by mDuo13** [comment]
Agreed the diagram should be changed. It should be an SVG for accessibility purposes and it shouldn't have the Ripple logo. Not sure how I missed that in my initial review

**Comment by oeggert** [comment]
I had the fix sitting in a draft PR, but pulled it out into a separate PR here in #2886

**Pull request merged**